### ISSUE TITLE
Updating a content type that has field's without default values causes an error

### ISSUE THREAD

**Issue by 99littlebugs** [contentful/contentful-management.py#116]
If i run: 
```python
myContentType = (
    contentful_management.Client(CONTENTFUL_MANAGEMENT_TOKEN, default_locale="en")
    .environments(CONTENTFUL_SPACE_ID)
    .find(environment)
    .content_types()
    .find("myContentType")
)
myContentType.save()
```

I get:
```
contentful_management.errors.UnprocessableEntityError: HTTP status code: 422
Message: Validation error
Details: 
	* Name: type - Path: '['fields', 0, 'defaultValue']' - Value: 'None'
```

I can get around this by getting the json value of this field, deleting the `defaultValue` field and then reapplying it.

**Comment by rubydog** [comment]
Hi @99littlebugs,

Thanks for reporting the issue.

I was able to reproduce the error but haven't identified the cause yet. I plan to investigate it within the next 2-3 weeks, assuming it's not urgent since you have a workaround. If it is urgent, please let me know, and I will prioritize this issue.

Cheers

**Comment by 99littlebugs** [comment]
when will this get released?

**Comment by rubydog** [comment]
@99littlebugs It has been released under version 2.14.2 https://github.com/contentful/contentful-management.py/blob/master/CHANGELOG.md
